semgrep/tests/e2e/targets/ignores/.semgrepignore

```diff
@@ -0,0 +1,24 @@
+# Ignore git items
```

Comment:
We should document this syntax (and the file in general) in this repo.

Reply:
Where in the repo should it be documented?

Reply:
It's currently documented in a comment on the semgrepignore Parser class, but it could be moved somewhere else

Reply:
Here's a PR to update the docs: https://github.com/returntocorp/semgrep-docs/pull/360